### ISSUE TITLE
utils的几个小问题

### ISSUE THREAD

**Issue by Greenhorntc** [tangxuemei1995/ancient-chinese-relation-calssificaion#1]
bert.py创建特征
all_e1_e2_mask = torch.tensor([f.e1_e2_mask for f in features], dtype=torch.long)  # add e1_e2 mask
all_first_pos_ids = torch.tensor([f.first_pos_ids for f in features], dtype=torch.long)
all_second_pos_ids = torch.tensor([f.second_pos_ids for f in features], dtype=torch.long)

这里看到utils.py创建features这里 convert_examples_to_features函数没这几个参数外
use_entity_type=config.use_entity_type, use_cnn=config.use_cnn,use_entity_position=config.use_entity_position)
e1_e2_mask,和 first_pos_ids以及all_second_pos_ids也没有对应的特征
